src/Microsoft.Azure.ServiceBus/Amqp/AmqpMessageSession.cs

```diff
@@ -8,7 +8,6 @@ namespace Microsoft.Azure.ServiceBus.Amqp
     using System.Threading.Tasks;
     using Azure.Amqp;
     using Core;
-    using Messaging.Amqp;
 
     class AmqpMessageSession : MessageSession
```

Comment:
Yes. More specifically, we need to expose a way to use "advanced functionality" for sessions as MessageReceiver doesn't handle sessions. 

---
In reply to: [113775192](https://github.com/Azure/azure-service-bus-dotnet/pull/146#discussion_r113775192) [](ancestors = 113775192)